src/lightning_app/core/queues.py

```diff
@@ -364,12 +364,11 @@ def get(self, timeout: int = None) -> Any:
 
         # timeout is some value - loop until the timeout is reached
         start_time = time.time()
-        timeout += 0.1  # add 0.1 seconds as a safe margin
         while (time.time() - start_time) < timeout:
             try:
                 return self._get()
             except queue.Empty:
-                time.sleep(HTTP_QUEUE_REFRESH_INTERVAL)
+                pass
```

Comment:
What's the effect of `time.seep` not being there anymore?

Reply:
This makes more call to the HTTP Server, but the flow shouldn't be blocked if no delta is available.
We were optimising to make the Queue general, but right now, this is pretty much internal.

For the Training Studio, without this change, we can't even support 5 experiments, but with it, we can support 50 per intermittence.

Reply:
right but we are ddos'ing the HTTP Server, we need a sleep of some kind or find another solution

Reply:
@lantiga In the current utlisation of this queue by the framework:
Here is the default condition: `(time.time() - start_time) < timeout`. This would be meet only once and therefore it won't loop over.

Secondly, this would mainly be replaced by get_all.

Reply:
alright, sounds ok since we are going for get_all